LICENSE

```diff
@@ -0,0 +1,10 @@
+By accessing NServiceBus code here, you are agreeing to the following licensing terms.
```

Comment:
This makes sense when read off NServiceBus repo, but not ServiceInsight and / or ServicePulse. If 'NServiceBus' referred to here means the company, could it be clarified? I mean, this is a license that people would need to read and understand clearly so why leave it vague?